packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -185,7 +183,16 @@ function (array $json) use (
                         $this->processPool->quitProcess($processIdentifier);
                         return;
                     }
-
+                    if (!isset($this->processRunCounter[$processIdentifier])) {
+                        $this->processRunCounter[$processIdentifier] = 0;
+                    }
+                    if ($this->processRunCounter[$processIdentifier] >= $this->maxProcessIterations) {
+                        $this->processPool->quitProcess($processIdentifier);
+                        if (is_callable($this->processSpawner)){
+                            ($this->processSpawner)();
+                        }
+                    }
+                    $this->processRunCounter[$processIdentifier] = $this->processRunCounter[$processIdentifier] + 1;
```

Comment:
```suggestion
                    ++$this->processRunCounter[$processIdentifier];
```

Reply:
is this one allowed by phpstan? having the ++ on the right side lead to a complaint

Reply:
I saw it used in the code base so I'd say yes it's allowed